burp/src/main/java/burp/BurpExtender.java

```diff
@@ -492,6 +492,7 @@ private JPanel buildDisplayPane()
         panelTitle.setHorizontalAlignment(SwingConstants.LEFT);
         JPanel basePanel = new JPanel();
         basePanel.setLayout(new BorderLayout());
+        displayArea.putClientProperty("html.disable", null);
```

Comment:
Note that client properties are removed when the UI changes. If Burp allows to dynamically change the L&F it will show the HTML after changing it.